crates/tspoint/src/lib.rs

```diff
@@ -0,0 +1,161 @@
+
```

Comment:
How about rustfmt the added files?

Reply:
That's actually a fair point.  There are no truly new files in this PR, but anything which has been sufficiently mangled after renaming will appear new once I squash it, so it's a good time to apply formatting.